app/javascript/components/bathroom.jsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Query } from 'react-apollo';
 import gql from 'graphql-tag';
 import styled from 'styled-components';
 import { WhiteSubTitle, WhiteTitleLarge } from '@components/typography';
 import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+// import withFragment from './hocs/with-fragment';
+import withFragmentModify from './hocs/with-fragment-modify';
 
-const getBathroomCode = gql`
-  query getBathroomCode($cityName: String!) {
-    location(cityName: $cityName) {
-      bathroomCode
-    }
+export const getBathroomCode = gql`
+  fragment Bathroom on Location {
```

Comment:
:thinking: Did we miss updating the bathroom code component in the last pass?

Reply:
I checked `master` and this change is already there. This might need a rebase

Reply:
Bathroom was updated in the last pass. A rebase is necessary.